infrastructure/terraform/to_review/notify-service.tf

```diff
@@ -1,131 +0,0 @@
-resource "aws_sqs_queue" "notify_irc_queue" {
```

Comment:
leave this

Reply:
fixed at https://github.com/sciurus/taskcluster/pull/14/files#diff-fbb1294efb073c6a507ffdd613837248R1